common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -809,7 +809,7 @@ public MergeStatuses finalizeShuffleMerge(FinalizeShuffleMerge msg) {
                 msg.shuffleMergeId, partition.reduceId);
             // This can throw IOException which will marks this shuffle partition as not merged.
             partition.finalizePartition();
-            if (partition.mapTracker.getCardinality() > 0) {
+            if (!partition.mapTracker.isEmpty()) {
```

Comment:
There should only be this one place where `getCardinality() > 0` is used for empty-checking on `RoaringBitmap `